model/model.mustache

```diff
@@ -73,25 +74,30 @@ module.exports = {
     if (!{{name}}.rev) {
       return Promise.reject(new httpError('BAD_REQUEST', 'No rev provided'));
     }
-    const valid = schema.validate({{name}});
-    if (!valid) throw new httpError('UNPROCESSABLE_ENTITY');
-    return r.table('{{pluralName}}')
-    .get({{name}}.id)
-    .update(
-      rethinkdb.ifRevMatches({{name}}),
-      { returnChanges: true }
-    )
-    .run()
-    .then(rethinkdb.checkRevError)
-    .then(rethinkdb.firstChange);
+
+    return r.table('{{pluralName}}').get({{name}}.id).run()
+    .then(current => {
```

Comment:
Yep, I think that's the go. As said in the doc below. That way it becomes really obvious how to remove individual properties from a doc, they're just not there in the doc you transmit.